product-manual/koala-agents.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "AI Agents"
+description: "Build a custom AI agent to automate your sales workflows"
+---
+
+## What is an agent?
+
+Agents are a way to automate your sales workflows. You can use them to simply answer questions about a profile or account, or to qualify leads and accounts.
+
+There are a few ways of creating an agent:
+- When setting up qualification in a play
+- AI agents on settings
+- Research agents
+
+Let's dive into each of these.
+
+## Qualification agents
+
+You can use a qualification agent to automatically qualify leads and accounts based on research. 
+
+### Creating an agent
+
+To create an agent you can use for qualification, select or create a new agent in the third step of the `Create New Play` flow:
+
+![Qualification agent](/images/product-manual/koala-agents/qualification.png)
+
+To create an AI agent, you'll need to:
+
+1. **Agent goal** - Define the purpose of your agent (required)
+2. **Question/Prompt** - Ask the specific question you want answered (required)
+3. **Research type** - Choose between "Companies" or "People" (required)
+4. **Research individual people** - Option for people-focused research
+5. **Response Type** - Select how the AI should format its answer
+6. **Advanced Options** - Additional configuration settings
+
+![Qualification agent](/images/product-manual/koala-agents/creating-ai-agent.png)
+
+For qualification agents, the **Response Type** is particularly important. Based on those you'll set up criteria for qualifying leads and accounts.
+
+### Qualification criteria and response types
+
+Take the following example:
+
+![Qualification agent](/images/product-manual/koala-agents/criteria-set-up.png)
+
+The agent named `Sales Contracts > $50k` has response type `Boolean`, which means that the agent will only return a `Yes` or `No` answer.
+
+You can then set up a rule to qualify a lead or account based on that answer.
+
+**Response types**
+
+We currently support the following response types:
+
+- **String**: A text response for open-ended answers
+- **Number**: A numerical value without currency formatting
+- **Currency**: A monetary amount with the appropriate currency symbol
+- **Percent**: A percentage value
+- **Boolean**: A Yes/No or True/False response
+- **Date**: A calendar date
+- **List**: Multiple items returned as a list
+- **Picklist**: Selection from predefined options
+
+Keep in mind that the response type will affect the criteria you can set up.
+
+## Research agents
+
+You can use AI agents to answer questions about a profile or account. Then you can show in a few places:
```

Comment:
Use "people" instead of the word "profiles". Customers dont know what a "profile" is (that's just our internal name for it in the database)